_rules/SC2-4-6-descriptive-headings.md

```diff
@@ -0,0 +1,134 @@
+---
+name: Descriptive headings
+test_type: atomic
+
+description: |
+   This rule checks that headings describe the topic or purpose of the content.
+   
+success_criterion:
+- 2.4.6 # Headings and labels
+
+test_aspects:
+- DOM Tree
+
+authors:
+- Dagfinn Rømen
+- Geir Sindre Fossøy
+- Carlos Duarte
+---
+
+## Test procedure
+
+### Applicability
+
+The rule applies to any element with the the [semantic role](#semantic-role) of `heading`.
+
+**Note**: The WCAG 2.0 success criterion 2.4.6 applies to all headings.
+
+**Note**: Heading content defines the header of a section (whether explicitly marked up using sectioning content elements, or implied by the heading content itself).
+
+### Expectation
+
+Each target element describes the topic or purpose of the subsequent content.
+
+**Note**: Headings do not need to be lengthy. A word, or even a single character, may suffice.
+
+## Assumptions
+
+Heading markup is covered by success criterion 1.3.1 Info and Relationships.
```

Comment:
This is not an assumption and should be removed. Maybe a note somewhere, but I'm not sure it is relevant.

Reply:
Agreed. Moved to applicability.

Reply:
Could you have another look @nitedog ?